test/common_methods_invocations.py

```diff
@@ -97,6 +97,7 @@ def __len__(self):
 M = 10
 S = 5
 
+
```

Comment:
Surprised this didn't cause lint to fail lol

Reply:
Because this wasn't a function before.